### PR TITLE
A few miscellaneous additions

### PR DESCRIPTION
**Pull request by MegaPiggy**
- Adds names to objects, textures. and materials that didn't have any.
- More Asteroid Customization
- Image Storage (so that they don't take up a ton of memory by repeatedly being created)
- Space Station Astro Object Type
- Adds the ability to stretch the mesh of a planet made with a height map
- Adds a few missing properties to schema

### CODE REVIEW

File: NewHorizons/External/HeightMapModule.cs [hunk 1]
```diff
@@ -12,5 +8,6 @@ public class HeightMapModule : Module
         public string TextureMap { get; set; }
         public float MinHeight { get; set; }
         public float MaxHeight { get; set; }
+        public Vector3 Stretch { get; set; } = Vector3.one;
```
Comment (Bwc9876) [inline NewHorizons/External/HeightMapModule.cs:11]: Should be `MVector3`, not `Vector3`

**Review by Bwc9876 — CHANGES_REQUESTED** [review]
Need to fix the type in HeightMapModule.cs

**Comment by Bwc9876** [comment]
Oh uh also might wanna change this to target `dev`

**Review by Bwc9876 — APPROVED** [review]

**Review by xen-42 — CHANGES_REQUESTED** [review]
Let's remove IsSpaceStation because it doesn't seem to do anything differently than IsSatellite, and because I want to remove IsSatellite in v1.0.0 when we refactor all the config stuff. Currently all it does differently than IsMoon is it removes the minimap when you get close to the body, so it'll just be replaced with something like "HasMinimap : false".

**Comment by MegaPiggy** [comment]
Done

**Comment by xen-42** [comment]
Gonna merge but I'm commenting out the texture caching stuff for now

**Pull request merged**